app/controllers/visualizations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class VisualizationsController < ApplicationController
+  def index
+    
+  end
+
+  def show
+  end
+
+  def new
+    @visualization = Visualization.new
+    2.times { @visualization.variables.build }
+  end
+
+  def create
+      @visualization = Visualization.new(visualization_params)
+      if @visualization.save
+         flash[:success] = "Visualization created!"
+         redirect_to visualizations_path
+      else
+         redirect_to new_visualization_path
+      end
```

Comment:
If we redirect to the new visualization path, would we just want to flash for now that there were errors bc the new visualization could not be added to the database?

Reply:
Sure, I can add that.

Reply:
Yes, we can add that after we merge